sdks/python/apache_beam/runners/common.py

```diff
@@ -58,23 +59,23 @@ def receive(self, windowed_value):
     raise NotImplementedError
 
 
-class DoFnMethodWrapper(object):
+class MethodWrapper(object):
   """For internal use only; no backwards-compatibility guarantees.
 
-  Represents a method of a DoFn object."""
+  Represents a method that can be invoked by `DoFnInvoker`."""
 
-  def __init__(self, do_fn, method_name):
+  def __init__(self, obj, method_name):
     """
-    Initiates a ``DoFnMethodWrapper``.
+    Initiates a ``MethodWrapper``.
 
     Args:
-      do_fn: A DoFn object that contains the method.
+      obj: the object that contains the method.
```

Comment:
<!--new_thread; commit:7ca389e990c4482eec47aecdc1913eeee0b169f9; resolved:0-->
Can you rename this and document what type of object this should be, and also validate this in the constructor if possible?  "obj" seems too generic